deepfence_server/model/registry.go

```diff
@@ -374,9 +374,9 @@ func ListImages(ctx context.Context, registryId string, filter reporters.FieldsF
 	}
 
 	query := `
-	MATCH (n:RegistryAccount{node_id: $id}) -[:HOSTS]-> (l:ImageStub) <-[:IS]- (m:ContainerImage)
-	` + reporters.ParseFieldFilters2CypherWhereConditions("l", mo.Some(filter), true) + `
-	RETURN l, m
```

Comment:
Why are we removing `:IS`?
I think this needs to stay since the `:IS` relationship allows extra data to be populated in the final object ended over by the rest API

Reply:
I am not sure why the ":IS" relationship is required in this case. 
Reason being, with the ":IS" query, we are getting the extra tags (the one which is retagged or duplicated from image of other repo). 
From what i see, we need to list all the images for the selected "repository" and "regsitry".  For that,  the above query should be sufficient (unless i am missing something).

Reply:
Hum.. I see why you made that change now - I still think we need keep the `:IS` relationship in, to be consistent across the board: all nodes with `:IS` relationship are done this way.
This actually raises a bigger question whether `ImageStub` now makes sense or not. So I would rather either remove `ImageStub` (and the `:IS` binding) entirely, or keep it. The one use case it solves (not sure we use it) is to retrieve all tags associated with a given name, so it still brings some value